pkgs/data/fonts/sarasa-gothic/default.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, p7zip}:
```

Comment:
Nitpick: Spaces before `stdenv` and after `p7zip` is the convention.

Reply:
Thanks for pointing these improvements, I'll do an update.